docs/user/reference/metrics/dual_labeled_counters.md

```diff
@@ -63,7 +49,7 @@ Glean.gleanUpload.failures["baseline"]["4xx"].add(3);
     These labels may contain any UTF-8 characters.
   * 4096 keys and 4096 categories if specified as static labels in `metrics.yaml`, see [keys](#keys) and [categories](#categories).
     Unknown keys and categories will be recorded under the special label `__other__`.
-    These labels may only be made of printable ASCII characters.
+    Each label may contain any utf-8 character _except_ for the ASCII record separator (0x1E).
```

Comment:
I think I was wrong here.
The static labels _are_ limited to ASCII characters.

The record separator part must go into the above dynamic labels point.